src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -284,6 +310,41 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         return email;
     }
     
+    private List<EmailWrapper> generateFeedbackSessionClosedEmail(
+            CourseAttributes course, FeedbackSessionAttributes session,
+            List<InstructorAttributes> instructors, List<StudentAttributes> students) {
+
+        List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
+
+        for (InstructorAttributes instructor : instructors) {
+            emails.add(feedbackSessionClosedEmail(course, session, instructor.name, instructor.email));
+        }
+        for (StudentAttributes student : students) {
+            emails.add(feedbackSessionClosedEmail(course, session, student.name, student.email));
```

Comment:
This method name should be `generateFeedbackSessionClosedEmail` because `feedbackSessionClosedEmail` suggests an object name, not a method name.